status base_status 
status arm_status 

